internal/app/machined/internal/server/v1alpha1/v1alpha1_server.go

```diff
@@ -1048,24 +1048,25 @@ func (s *Server) Processes(ctx context.Context, in *empty.Empty) (reply *machine
 	)
 
 	for _, proc := range procs {
+		// due to race condition, reading process info might fail if process has already terminated
 		command, err = proc.Comm()
 		if err != nil {
-			return nil, err
+			continue
```

Comment:
this should fix that bug with empty output from `talosctl processes`, not perfect, but I don't know better way